pymc3/tests/test_distributions.py

```diff
@@ -1893,8 +1893,7 @@ def test_mvnormal_init_fail(self):
             with pytest.raises(ValueError):
                 x = MvNormal("x", mu=np.zeros(3), cov=np.eye(3), tau=np.eye(3), size=3)
 
-    @pytest.mark.parametrize("n", [1, 2, 3])
-    @pytest.mark.xfail(reason="Distribution not refactored yet")
+    @pytest.mark.parametrize("n", [2, 3])
```

Comment:
This fails for n=1 , maybe the dimensions are being downcasted to a scalar when matrix is shaped `1x1`.

Reply:
If that `n` is being used to determine the number of output dimensions, then `n=1` might not make sense for a matrix-valued distribution like this.

Reply:
I raised an error if such a matrix is passed.